index.html

```diff
@@ -45,77 +44,148 @@ <h1 class="heading">TheYalow</h1>
     </header>
     <section class="slider">
       <div class="slider-wrapper">
-      <div class="video-conrtainer"><video src="https://www.youtube.com/watch?v=RaQp6oM-8_8" poster="assets/poster.jpg" class="video-slider"></video></div>
-      <div class="title-slider">
-        <p class="slider-title"><span class="slider-title-main">The cool</span></br>The minimal</p>
-      </div>
+
+        <div class="video-conrtainer-1300">
+          <video src="https://www.youtube.com/watch?v=RaQp6oM-8_8" poster="assets/poster.jpg" class="video-slider"></video>
```

Comment:
_"Video element placed with image placeholder."_
You have no placeholder
https://stackoverflow.com/questions/14616453/image-placeholder-fallback-for-html5-video